bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -81,7 +89,7 @@ struct {
 
 SEC(".rodata.config")
 __attribute__((
-	btf_decl_tag("Sample Rate"))) static volatile const int SAMPLE_RATE = 5;
```

Comment:
Could you put some comment here where this number can be overwrote?

Reply:
The sampling rate should be disabled by default since we don't know the impact in the measurement.

Reply:
I agree to have this disabled and set to zero. Thank you for adding the comment above.

Reply:
What @vimalk78 said. The number here is irrelevant as it's set from userland. There was previously an issue where this got optimized out when it was `0` but since it changed to be `static volatile const` that shouldn't happen in future.

The comment should be updated to reflect the current state.

Reply:
That is right, but just in case it is better to set the default value here to 0.